google/async_resumable_media/requests/download.py

```diff
@@ -465,18 +431,19 @@ def _add_decoder(response_raw, checksum):
         patched, or a ``_DoNothingHash`` if the decoder is patched, since the
         caller will no longer need to hash to decoded bytes.
     """
+
     encoding = response_raw.headers.get(u"content-encoding", u"").lower()
     if encoding != u"gzip":
         return checksum
 
     response_raw._decoder = _GzipDecoder(checksum)
-    return _helpers._DoNothingHash()
+    return _helpers._DoNothingHash
 
 
 class _GzipDecoder(urllib3.response.GzipDecoder):
     """Custom subclass of ``urllib3`` decoder for ``gzip``-ed bytes.
 
-    Allows a checksum function to see the compressed bytes before they are
+    Allows a checksum hash function to see the compressed bytes before they are
```

Comment:
No reason to change this; technically crc32c is not a hash.

Reply:
Guessing this is a merge issue. dropped.